invenio_rdm_records/services/review/service.py

```diff
@@ -190,5 +199,17 @@ def submit(self, identity, id_, data=None, require_review=False, uow=None):
                 identity, community, request, uow
             )
 
+            uow.register(
+                NotificationOp(
+                    CommunityInclusionNotificationBuilder.build(request_item)
+                )
+            )
```

Comment:
```suggestion
            uow.register(
                NotificationOp('community-submission', request=request_item._request)
            )
```

minor/shelve: maybe we could simplify this call to something like this? let's shelve and revisit maybe after we have the full example integrated.